dev-db/mysql-connector-c/files/mysql-connector-c-8.0.27-res_n.patch

```diff
@@ -0,0 +1,47 @@
+# Musl doesn't have res_n* functions so we are falling back to the not
+# thread safe ones. Patch made with help from developer Fabian Groffen
+# <grobian@gentoo.org>.
+#
+# Closes: https://bugs.gentoo.org/761352
```

Comment:
I'm going to throw in the upstream link even though it's not the same exact patch. Good work!